yt_dlp/extractor/bilibili.py

```diff
@@ -666,14 +667,14 @@ class BiliBiliBangumiIE(BilibiliBaseIE):
 
     def _real_extract(self, url):
         episode_id = self._match_id(url)
-        webpage = self._download_webpage(url, episode_id)
+        headers = {'Referer': url, **self.geo_verification_headers()}
+        webpage = self._download_webpage(url, episode_id, headers=headers)
```

Comment:
since it doesn't make much sense to pass a self-referential referer header, I would to this (with the below suggestion too)
```suggestion
        headers = self.geo_verification_headers()
        webpage = self._download_webpage(url, episode_id, headers=headers)
```